README.md

```diff
@@ -261,7 +275,7 @@ When preparing a pull request, if possible please configure with
 with:
 
 ```sh
-make CFLAGS="-Wall -Wextra -Werror"
+make CFLAGS="-Wall -Wextra -Wshadow -Werror"
```

Comment:
I think that the entire "Contributing" should be moved to the CONTRIBUTING.md and in the README we should keep only a reference like: "See the [CONTRIBUTING.md file](CONTRIBUTING.md) for contribution guidelines." Having a separate contributing file will align better with GitHub Community Standards (https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)